### ISSUE TITLE
Color theme

### ISSUE THREAD

**Issue by theis999** [MortenTobiasNielsen/Factorio-GUI-V2#259]
Zaspar has requested a dark mode. It is however not possible to use dark mode in wxWidget currently, due to some limitations of the windows api (or something).

We could however consider changing the theme of the application to be more night friendly

**Comment by MortenTobiasNielsen** [comment]
It would be really nice if the default theme was something like dark mode.

**Comment by theis999** [comment]
It is a shame that the vcpkg version is 0.1 behind. As there is some experimental dark mode in 3.3.0 of wx: https://docs.wxwidgets.org/trunk/classwx_app.html#af8c93d7e3345e62a58325f3ab1d158d6

**Comment by MortenTobiasNielsen** [comment]
Do you have a feeling for how long it takes for something in experimental to get into the normal version? I guess the answer for this type of project is "a long time". :)

**Comment by theis999** [comment]
I have no idea. But a long time seems like fair assessment 😅

Unfortunately, setting darker colors on the gui is only half-possible. As the borders don't change color and control-elements like the number spinners also don't react to styling.
And i am not particularly skilled at graphic design

**Comment by MortenTobiasNielsen** [comment]
Yeah that doesn't sound optimal. Yeah graphical design is rally not my thing either. 🙂 Do you want to bump the version and try out the experimental feature?

**Comment by theis999** [comment]
It'd be fun to see, however it would require abandoning vcpkg which means a substantiel change to building the project. So that probably isn't easy.

**Comment by MortenTobiasNielsen** [comment]
I don't have a feeling for the complexity of doing this - but it doesn't sound easy.

**Comment by theis999** [comment]
https://www.wxwidgets.org/develop/roadmap/
![image](https://github.com/MortenTobiasNielsen/Factorio-TAS-Generator/assets/18309265/dce7144a-3ac3-456c-8494-5e58d9ce09bb)

It looks like 3.3 is still some ways away.
